src/implementation/Client/GRPCClient/configuration.ts

```diff
@@ -111,18 +118,23 @@ export default class GRPCClientConfiguration implements IClientConfiguration {
 
     stream.on("data", async (data: SubscribeConfigurationResponse) => {
       streamId = data.getId();
+      const configMap = Object()
+      data.getItemsMap().forEach(function(v, k) {
+        const item: ConfigurationItem = {
+          key : k,
+          value : v.getValue(),
+          version : v.getVersion(),         
+          metadata :  v.getMetadataMap().toObject().reduce((result: object, [key, value]) => {
```

Comment:
The conversion from gRPC metadata to KeyValuePair, shall we move this to Client.utils.ts and reuse the same in both places?